### PR TITLE
Fixed issue #271

### PR DESCRIPTION
**Pull request by TheSamSmith**
Fix for issue #271 so that identityAddress can be accessed in the userData object as part of Hacktoberfest

### CODE REVIEW

**Comment by TheSamSmith** [comment]
Its not letting me sign the CLA?

**Comment by TheSamSmith** [comment]
@larrysalibra should be good to merge now

**Comment by jackzampolin** [comment]
Looks like you are good on the CLA! The bot just needed a good kick.

File: src/auth/authApp.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ export function handlePendingSignIn(nameLookupURL: string = 'https://core.blocks
           profile: tokenPayload.profile,
           appPrivateKey: tokenPayload.private_key,
           coreSessionToken: tokenPayload.core_token,
+          identityAddress: tokenPayload.iss,
```
Comment (larrysalibra) [inline src/auth/authApp.js:138]: `tokenPayload.iss` isn't the identity address - it is the decentralized identifier. The identity address needs to be parsed from that.

**Review by larrysalibra — CHANGES_REQUESTED** [review]

File: src/auth/authApp.js (hunk 1 above)
Comment (vsund) [reply src/auth/authApp.js:138]: I guess this should be only a call to [`getAddressFromDID(did)`](https://github.com/blockstack/blockstack.js/blob/develop/src/dids.js#L25) around `tokenPayload.iss`, correct @larrysalibra?

**Comment by larrysalibra** [comment]
This was resolved by @bitsphr in https://github.com/blockstack/blockstack.js/pull/391#issuecomment-378228337

Thanks @TheSamSmith for getting the ball rolling on this!

**Pull request closed without merging**